test/utils/socketErrorMessages.test.js

```diff
@@ -122,6 +122,16 @@ describe('Socket error messages', () => {
 			[basePayload('illegalButton'), 'Specified buttons are not supported on this device. Chain description'],
 			[basePayload('unsupportedButton'), 'Specified buttons are not supported on this device. Chain description'],
 			[basePayload('aborted'), 'Test execution was aborted. Chain description'],
+			[
+				{
+					...basePayload('aborted'),
+					response: {
+						...basePayload('aborted').response,
+						message: {info: {reason: 'manualActionRequired'}},
+					},
+				},
```

Comment:
For consistency you can replace it to something:
`
set(lensPath(['response', 'message', 'info', 'reason']), 'manualActionRequired', basePayload('aborted')),
`

Reply:
updated